File: Trabalho de py e java 3.0/Dados.py
from datetime import date
Nome=input("Digite seu nome: ")
idade=int(input("Digite sua idade:"))
data_nasc_str = input("Digite sua data de nascimento (DD/MM/AAAA): ")
dia, mes, ano = map(int, data_nasc_str.split('/'))
#########################################3
from datetime import datetime
agora = datetime.now()
hora = agora.hour
minuto = agora.minute
if 5 <= hora < 12:
    print("Bom dia!!")
elif 12 <= hora < 18:
    print("Boa tarde.")
else:
 print("Boa noite")

if idade >= 18:
        print(
              
f"!Entrada liberada para o barzinho :D!")
else:
  print(
f"!Pode não,vagabundo >:(!")

File: Trabalho de py e java 3.0/Dados.java
import java.time.LocalDate;
import java.time.LocalTime;
import java.util.Scanner;
public class Dados{
    public static void main(String[]args){
        Scanner scanner=new Scanner(System.in);
    System.out.print("Digite seu nome: ");
    String Nome=scanner.nextLine();
    System.out.print("Digite sua idade: ");
    int Idade=scanner.nextInt();
////vontade de dormir slk
System.out.print("Dia (dd): ");
        int dia = scanner.nextInt();
        System.out.print("Mês (mm): ");
        int mes = scanner.nextInt();
        System.out.print("Ano (aaaa): ");
        int ano = scanner.nextInt();
    LocalDate Nascimento = LocalDate.of(ano, mes, dia);
    System.out.println("Data de nascimento: " + Nascimento);
////////////////////////////////////////////////////
    LocalTime agora=LocalTime.now();
if (agora.getHour() > 5 && agora.getHour() < 12) {System.out.println("Bom dia.");} 
else if (agora.getHour() > 12 && agora.getHour() < 18) {System.out.println( "Boa tarde.");}
else{System.out.println("Boa noite.");}
///////////////////////////////////////////////////
if (Idade >= 18){System.out.println("Pode entrar no Bar.");}
else{System.out.println("Sai daqui,pirralho(a)!");}

    scanner.close();

    }
}